docs/Deployed_Contracts/Monad/monad_test.md

```diff
@@ -0,0 +1,15 @@
+# Monad Testnet
+
+| Name                            | Address                                                                                      | Interface |
+|---------------------------------|----------------------------------------------------------------------------------------------| --------- |
+| Ammalgam Factory               | [`0x83f3...6699`](https://sepolia.etherscan.io/address/0x83f33E2fd340d2beAC496473002389F458b06699) | [IAmmalgamFactory](https://github.com/Ammalgam-Protocol/core-v1/blob/8a7f458eaa44bd6bb81314db98899ee7d35f8c57/contracts/interfaces/factories/IAmmalgamFactory.sol)
```

Comment:
These are Sepolia address. Lets add this page later when we have the Monad Addresses.